docs/actions-pipelines.md

```diff
@@ -5,7 +5,7 @@ This section covers how to develop, run, and test your code to ensure it will wo
 
 ## Project pipelines
 
-The [cohortextractor](actions-cohortextractor.md) section describes how to make an action which generate dummy datasets based on the instructions [defined in your `study_definition.py` script](study-def.md).
+The [ehrQL](/ehrql/) documentation describes how to make an action which generate dummy datasets based on the instructions defined in your `dataset_definition.py` script.
```

Comment:
thought: There's also a slight quirk here where, previously, `study_definition.py` encompassed measures or what we now call datasets, whereas `dataset_definition.py` explicitly refers to datasets (and presumably someone would use `measure_definition.py` instead for measures).

I don't think probably matters too much though.

Reply:
I don't think these docs are tight enough to make this worth worrying about.